The paths to MAKER executables set by MAKER are already correct, avoid
tampering with them.
author: Alex Domingo (Vrije Universiteit Brussel)
--- run_maker.py.orig	2023-03-21 09:19:05.057679000 +0100
+++ run_maker.py	2023-03-21 09:19:49.143408491 +0100
@@ -463,14 +463,6 @@
     replace('maker_opts.ctl', 'protein_pass=0', 'protein_pass=1')
     replace('maker_opts.ctl', 'rm_pass=0', 'rm_pass=1')
 
-    # Program paths
-    for program in [
-            'makeblastdb', 'blastn', 'blastx', 'tblastx', 'RepeatMasker',
-            'exonerate', 'snap', 'augustus', 'tRNAscan-SE', 'snoscan']:
-        replace('maker_exe.ctl', '{}='.format(program), '{}={}'.format(
-            program, os.path.join(os.path.dirname(maker_bin), program)
-        ))
-
     # Last run, keep_preds=1
     if version == '4':
         replace('maker_opts.ctl', 'keep_preds=0', 'keep_preds=1')
